ci/pipeline.yml

```diff
@@ -0,0 +1,82 @@
+# COMMON ========================================
+
+aws_secret_access_key:
+  &aws_secret_access_key ((aws.service_account_access_key_secret))
+aws_access_key_id:
+  &aws_access_key_id ((aws.service_account_access_key_id))
+
+aws_params: &aws_params
+  AWS_ACCESS_KEY_ID: *aws_access_key_id
+  AWS_SECRET_ACCESS_KEY: *aws_secret_access_key
+  AWS_REGION: ((aws_region))
+  AWS_SERVICE_ROLE: arn:aws:iam::((aws.account_((environment)))):role/bpm-svc-deploy-mail-lambda
+  S3_NAME: ((environment)).bpm.((aws.tf_state_s3))
+  S3_KEY: lambdas/mail.tfstate
```

Comment:
May be better keyed by name of service first (e.g. `mail-receiver/lambda.tfstate` or similar)

Reply:
How many dependencies there if I make that change - would the time spent on that be reasonable?

Reply:
If there's nothing already deployed it won't make a difference. So if say there's already a pipeline up, do a teardown, change this, then refly/deploy :)

Reply:
Done